chapters/inheritance.tex

```diff
@@ -1069,6 +1071,10 @@ \subsection{Annotations for Redeclaration and Modification}\label{annotation-cho
 
 This is a hint for users of the model, and can also be used by the user interface to suggest reasonable redeclaration, where the string comments on the choice declaration can be used as textual explanations of the choices.
 The annotation is not restricted to replaceable elements but can also be applied to non-replaceable elements, enumeration types, and simple variables.
+Some choices may be invalid in the current context, e.g., a value might violate the min-attribute.
+\begin{nonnormative}
+Tools may avoid including such choices, mark them as invalid, or detect such violations later.
+\end{nonnormative}
```

Comment:
I believe readability would benefit from including the remark in the normative paragraph.
```suggestion
(Options for tools encountering such choices include not showing them, marking them as invalid, or detecting the violations later.)

```